user-manual/transfer/clean-up-names.rst

```diff
@@ -0,0 +1,91 @@
+.. _clean-up-names:
+
+==============
+Clean up names
+==============
+
+The clean up names microservice runs twice during processing - once on the
+Transfer tab and once on the Ingest tab. The purpose of this microservice is to
+ensure that Archivematica's tools and processes do not fail because of
+characters that appear in file or directory names.
+
+Archivematica groups a wide variety of tools together to create preservation
+workflows. Some of the tools that Archivematica uses have a narrowly-defined
+scope in terms of what the tool considers to be a valid character for a file or
+directory name. Encountering a character that the tool considers to be invalid
+can cause the tool to fail, which can halt processing and interfere with normal
+operation of Archivematica.
+
+To prevent these kinds of tool failures from happening, Archivematica implements
+a script that changes file and directory names to conform to requirements of the
+most restrictive tools. Valid characters are defined by `specific code`_ in
```

Comment:
> Valid characters are defined by specific code in Archivematica:

I see what you mean. There might be a way we can be more precise for the reader here, not just to help us maintain the sensitivity with which you've approached these docs, but to open up a greater possibility of change as well.

If we say:

`Valid characters are defined by specific code used by Archivematica:`

The context is that outside of these docs we can better clarify that the limited set below is not entirely a design choice by our team. The fact that these characters is *such* as narrow band comes from the standard Python library we use: https://docs.python.org/3/library/string.html#string.ascii_lowercase 

We could add the detail, but I'm only advocating for `in` to be replaced with `used by`.

Reply:
I think this is a good suggestion! Can you recommend a line change? I want to make use of this fun feature :smile_cat:

Reply:
Hahaha! I feel like I'm being a party-pooper, but because my comment and first suggestion are restricted to the one line it won't let me! Sorry :smiley:

Reply:
Hehe that's okay. Thanks.

Reply:
Fixed!